ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml

```diff
@@ -65,7 +65,7 @@ resources:
       memory: 200Mi
 tests:
 - as: aws-c2s-ipi-disconnected-private-p2-f14
-  cron: 34 11 5,20 * *
+  cron: 28 23 8,23 * *
```

Comment:
@jianlinliu Any update to the script ?
This  is what i got,
```
[lxia@homepc release]$ ./ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh aws-c2s-ipi-disconnected-private-p2-f14 openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml
cron: 34 11 5,20 * *
```

Reply:
Interesting, you are right, let me update it again.

Reply:
Okay, I found the root cause, I gave a full path for yaml file, so generated different cron entries.

Now they are fixed, pls help review.